test/integration/executor_test.go

```diff
@@ -209,4 +209,160 @@ var _ = ginkgo.Describe("ManifestWork Executor Subject", func() {
+				Subject: workapiv1.ManifestWorkExecutorSubject{
+					Type: workapiv1.ExecutorSubjectTypeServiceAccount,
+					ServiceAccount: &workapiv1.ManifestWorkSubjectServiceAccount{
+						Namespace: o.SpokeClusterName,
+						Name:      executorName,
+					},
+				},
+			}
+		})
+
+		ginkgo.It("no permission", func() {
+			roleName := "role1"
+			_, err = spokeKubeClient.RbacV1().ClusterRoles().Create(
+				context.TODO(), &rbacv1.ClusterRole{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: roleName,
+					},
+					Rules: []rbacv1.PolicyRule{
+						{
+							// no "escalate" and "bind" verb
+							Verbs:     []string{"create", "update", "patch", "get", "list", "delete"},
+							APIGroups: []string{"rbac.authorization.k8s.io"},
+							Resources: []string{"clusterroles", "clusterrolebindings"},
+						},
+					},
+				}, metav1.CreateOptions{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			_, err = spokeKubeClient.RbacV1().ClusterRoleBindings().Create(
+				context.TODO(), &rbacv1.ClusterRoleBinding{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: o.SpokeClusterName,
+						Name:      roleName,
+					},
+					Subjects: []rbacv1.Subject{
+						{
+							Kind:      "ServiceAccount",
+							Namespace: o.SpokeClusterName,
+							Name:      executorName,
+						},
+					},
+					RoleRef: rbacv1.RoleRef{
+						APIGroup: "rbac.authorization.k8s.io",
+						Kind:     "ClusterRole",
+						Name:     roleName,
+					},
+				}, metav1.CreateOptions{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			work, err = hubWorkClient.WorkV1().ManifestWorks(o.SpokeClusterName).Create(
+				context.Background(), work, metav1.CreateOptions{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			util.AssertWorkCondition(work.Namespace, work.Name, hubWorkClient, string(workapiv1.WorkApplied),
+				metav1.ConditionFalse,
+				[]metav1.ConditionStatus{metav1.ConditionFalse, metav1.ConditionFalse, metav1.ConditionFalse},
+				eventuallyTimeout, eventuallyInterval)
+			util.AssertWorkCondition(work.Namespace, work.Name, hubWorkClient, string(workapiv1.WorkAvailable),
+				metav1.ConditionFalse,
+				[]metav1.ConditionStatus{metav1.ConditionFalse, metav1.ConditionFalse, metav1.ConditionFalse},
+				eventuallyTimeout, eventuallyInterval)
+
+			// ensure configmap not exist
+			util.AssertNonexistenceOfConfigMaps(
+				[]workapiv1.Manifest{
+					util.ToManifest(util.NewConfigmap(o.SpokeClusterName, "cm1", map[string]string{"a": "b"}, []string{})),
+				}, spokeKubeClient, eventuallyTimeout, eventuallyInterval)
+		})
+
+		ginkgo.It("with permission", func() {
+			roleName := "role1"
+			_, err = spokeKubeClient.RbacV1().ClusterRoles().Create(
+				context.TODO(), &rbacv1.ClusterRole{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: roleName,
+					},
+					Rules: []rbacv1.PolicyRule{
+						{
+							// with "escalate" verb
+							// Verbs: []string{"create", "update", "patch", "get", "list", "delete", "escalate"},
```

Comment:
it is weird that `Verbs: []string{"create", "update", "patch", "get", "list", "delete", "escalate"},` does not work, but `Verbs: []string{"*"},` does